geniza/common/management/commands/visual_tests.py

```diff
@@ -40,50 +46,51 @@ def take_snapshots(self, browser):
         browser.find_element_by_css_selector(
             ".doctype-filter li:nth-child(1) label"
         ).click()
-        percy_snapshot(browser, "Document Search filter")
+        percy_snapshot(browser, "Document Search filter%s" % dark_mode_str)
 
         # document search
         browser.get(
             "http://localhost:8000/en/documents/?q=the+writer+Avraham+באנפנא&per_page=2"
         )
-        percy_snapshot(browser, "Document Search")
+        percy_snapshot(browser, "Document Search%s" % dark_mode_str)
 
         # document detail
         browser.get("http://localhost:8000/en/documents/2532/")
-        percy_snapshot(browser, "Document Details")
+        percy_snapshot(browser, "Document Details%s" % dark_mode_str)
 
         # document scholarship
         browser.get("http://localhost:8000/en/documents/9469/scholarship/")
-        percy_snapshot(browser, "Document Scholarship Records")
+        percy_snapshot(browser, "Document Scholarship Records%s" % dark_mode_str)
 
         # mobile menu
         browser.get("http://localhost:8000/en/documents/2532/#menu")
+        mobile_menu_css = "ul#menu { left: 0 !important; transition: none !important; }"
         percy_snapshot(
-            browser,
-            "Mobile menu",
-            percy_css="ul#menu { left: 0 !important; transition: none !important; }",
+            browser, "Mobile menu%s" % dark_mode_str, percy_css=mobile_menu_css
         )
 
         # about submenu open on both desktop and mobile
         browser.get("http://localhost:8000/en/documents/2532/#menu")
         # open about menu
         browser.find_element_by_id("open-about-menu").send_keys(Keys.ENTER)
+        about_menu_css = "ul#about-menu { left: 0 !important; transition: none !important; } @media (min-width: 900px) { ul#about-menu { left: auto !important; } }"
```

Comment:
remind me, why do we need custom css for this?

Reply:
Added some comments to explain this:

```python
# custom CSS to ensure that on mobile, the menu transition is disabled and the menu
# is in the correct position
mobile_menu_css = "ul#menu { left: 0 !important; transition: none !important; }"
```

```python
# custom CSS to ensure that on mobile, the about menu transition is disabled and the menu
# is in the correct position; and that on desktop, that override does not impact its position
about_menu_css = "ul#about-menu { left: 0 !important; transition: none !important; } @media (min-width: 900px) { ul#about-menu { left: auto !important; } }"
```

Reply:
oh right! the transitions. I don't love that you're overriding positioning as well, but I guess we can live with it. Thanks for adding comments.

Reply:
I don't love it either! 😆 I think the reason I did this is that the initial position for those menus is `+/-100vh` when you visit `#menu` or `#about-menu`, so they'll appear offscreen when Percy takes the screenshot. The transition moves them over, but since we disable the transition for Percy, we also have to move them manually. There's probably a better way to do this, though…

Reply:
ok. well, the comment should help for now until we can figure out a better solution.